cmd/sync.go

```diff
@@ -57,4 +72,7 @@ func init() {
 	syncCmd.Flags().BoolVar(&dumpConfig.SkipConsumers, "skip-consumers",
 		false, "do not diff consumers or "+
 			"any plugins associated with consumers")
+	syncCmd.Flags().BoolVarP(&syncCmdKongStateFileFromStdin, "read-state-from-stdin",
+		"-", false, "read Kong's configuration state from stdin (overrides "+
+			"--state flag)")
```

Comment:
Please remove this flag. `deck sync -s -` should be good end user experience for reading the content from stdin or writing it to stdout.